mailmerge/__main__.py

```diff
@@ -71,13 +71,13 @@
     "--config", "config_path",
     default="mailmerge_server.conf",
     type=click.Path(),
-    help="server configuration (mailmerge_server.conf)",
+    help="SMTP configuration (mailmerge_server.conf)",
 )
 @click.option(
     "--output-format", "output_format",
     default="colorized",
     type=click.Choice(["colorized", "text", "raw"]),
-    help="Output format (colorized).",
+    help="Logging output format (colorized).",
```

Comment:
I would prefer the original.  There's a plan to add logging which is different.

Reply:
From my perspective, *the e-mails* are the real output of the program, and I honestly didn't expect the console output to be configurable. I didn't know about the [csplit trick](https://github.com/awdeorio/mailmerge/pull/107#issuecomment-720706463) that allows one to collect the e-mails from stdout.
Maybe: "Console output format (colorized)"?

Reply:
"Console output format" sounds good